meta-jsdelivr/recipes-jsdelivr/jsdelivr-scripts/files/jsdelivr-startWorld.sh

```diff
@@ -59,7 +60,7 @@ while [ 1 ]; do
     RUNNING=$(docker inspect --format='{{.State.Running}}' globalping-probe)
 
     if [ "$RUNNING" != "true" ]; then
-        /usr/bin/docker run -d --network host --restart=always --name globalping-probe ghcr.io/jsdelivr/globalping-probe:latest
```

Comment:
This change is wrong. `ghcr.io/jsdelivr/globalping-probe` is the correct registry we use. I dont think `jsdelivr/globalping-probe` even exists.

Also are you sure you dont want to run in detached mode?

Reply:
Regarding the detached mode yep, it was a typo...

Regarding the registry name, that's referring to the local registry that resides in RAM, not to the one located in the exterior

Reply:
But it's a regular docker command. Won't it assume by default it's pointing to docker hub for the repo?

Reply:
Yes, but the way the container image is captured(downloaded+compressed),  the server name is dropped because the image will be "local."

And when the image is loaded in RAM, doesn't contain the server name portion, and because of that, the docker command is that runs that image as a container doesn't refer to the image with the server name

PS: The container update process uses the full name(with the server) of course